tests/python_test.py

```diff
@@ -487,8 +504,94 @@ def test_write_resource_invalid_disk(tmpdir, sanlock_daemon, disk):
         sanlock.write_resource("ls_name", "res_name", disks)
     assert repr(disk) in str(e.value)
 
+
 @pytest.mark.parametrize("filename,encoding", FILE_NAMES)
 def test_killpath(tmpdir, sanlock_daemon, filename, encoding):
     cmd_path = util.generate_path(tmpdir, filename, encoding)
     fd = sanlock.register()
     sanlock.killpath(cmd_path, [cmd_path], fd)
+
+
+@contextmanager
+def raises_sanlock_errno(expected_errno=errno.ECONNREFUSED):
+    with pytest.raises(sanlock.SanlockException) as e:
+        yield
+    assert e.value.errno == expected_errno
+
+
+@pytest.mark.parametrize("name", LOCKSPACE_OR_RESOURCE_NAMES)
+def test_rem_lockspace_parse_args(name):
+    assert not util.sanlock_is_running(), "Test expects sanlock to be down"
```

Comment:
This will cause the test to fail, instead of error.

Why not create a no_sanlock_daemon fixture, raising SanlockIsRunning exception
if util.sanlock_is_running() returns True?

    @pytest.mark.parametrize("name", LOCKSPACE_OR_RESOURCE_NAMES)
    def test_rem_lockspace_parse_args(no_sanlock_daemon, name):
        ...

The current code duplicate the assert and the text for every test.